lib/ae_mdw_web/benchmark/bench.ex

```diff
@@ -92,8 +92,6 @@ defmodule Mix.Tasks.Bench do
 
   defp percentile([], _), do: nil
   defp percentile([x], _), do: x
-  defp percentile(list, 0), do: min(list)
-  defp percentile(list, 100), do: max(list)
```

Comment:
maybe leave this optimization to avoid sorting all.

Reply:
Dialyzer was complaining about these two clauses because they weren't being used

Reply:
Indeed